portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,185 @@
+          <p> In this project, I attempted to create a computational model capable of classifying an image of a stained white blood cell into one of the four major types of white blood cells. Specifically, I used Python to preprocess a set of publicly available training images before using Tensorflow and Keras to create, train, and test three convolutional neural networks with the best model achieving an accuracy of 78%.   </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="w3-col m4 w3-margin-bottom">
+      <div class="w3-light-grey">
+        <img src="images/array_bounds_example.png" alt="array" style="width:100%">
+        <div class="w3-container">
+          <h4>Range Analysis for Array Bounds Checking in Java</h4>
+          <p class="w3-opacity">Program Analysis - Spring 2020 &nbsp <a href="https://github.com/PriyaVarra/Context-Sensitive-Interprocedural-Range-Analysis-for-Java"> <img src="images/GitHub-Mark-32px.png" width="32" height="32"> </a></p>
+          <p> In this project, I designed and implemented a context-sensitive interprocedural dataflow analysis that tracks the possible values of integer variables during program execution for Jimple -an intermediate representation of Java bytecode provided by the Soot framework. 
+              I utilized this analysis to provide errors and warnings about negative and out of bounds array accesses for Java code samples. </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <!-- Hobbies -->
+  <div class="w3-container" id="hobbies" style="margin-top:75px">
+    <h1 class="w3-xxxlarge w3-text-teal"><b>Hobbies</b></h1>
+    <hr style="width:50px;border:5px solid teal" class="w3-round">
+    <p>Do you want us to style your home? Fill out the form and fill me in with the details :) We love meeting new people!</p>
+    <form action="/action_page.php" target="_blank">
+      <div class="w3-section">
+        <label>Name</label>
+        <input class="w3-input w3-border" type="text" name="Name" required>
+      </div>
+      <div class="w3-section">
+        <label>Email</label>
+        <input class="w3-input w3-border" type="text" name="Email" required>
+      </div>
+      <div class="w3-section">
+        <label>Message</label>
+        <input class="w3-input w3-border" type="text" name="Message" required>
+      </div>
+      <button type="submit" class="w3-button w3-block w3-padding-large w3-teal w3-margin-bottom">Send Message</button>
+    </form>  
+  </div>
+
+  <!-- Gallery -->
+  <div class="w3-container" style="margin-top:80px" id="gallery">
+    <h1 class="w3-xxxlarge w3-text-teal"><b>Gallery</b></h1>
+    <hr style="width:50px;border:5px solid teal" class="w3-round">
+  </div>
+  
+
+  <div class="w3-row-padding">
+    <div class="w3-half">
+      <img src="images/myAvatar.png" style="width:100%" onclick="onClick(this)" alt="Concrete meets bricks">
```

Comment:
Not necessary but it is always nice to break up long lines: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
Thanks for the suggestion! I will check this out in my most recent version.